raiutils/raiutils/models/models.py

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) Microsoft Corporation
+# Licensed under the MIT License.
+
+
+class SKLearn(object):
+    """Provide scikit-learn related constants."""
+
+    EXAMPLES = 'examples'
+    LABELS = 'labels'
+    PREDICT = 'predict'
+    PREDICTIONS = 'predictions'
+    PREDICT_PROBA = 'predict_proba'
+
+
+def is_classifier(model):
```

Comment:
I'm not completely sure, but you may need to add this to raiutils first, merge that PR, publish it, and then make a separate PR for raiwidgets to use this method

Reply:
done.